frontend/micro-ui/web/micro-ui-internals/packages/Mukta/src/services/Search.js

```diff
@@ -0,0 +1,114 @@
+import _ from "lodash";
+
+const createProjectsArray = (t, project, searchParams, headerLocale) => {
+    let totalProjects = {
+        searchedProject : {},
+        subProjects : []
+    };
+    let basicDetails = {};
+    let totalProjectsLength = project.length;
+    for(let projectIndex = 0; projectIndex < totalProjectsLength; projectIndex++) {
+        let currentProject = project[projectIndex];
+        const headerDetails = {
+            title: " ",
+            asSectionHeader: true,
+            values: [
+                { title: "WORKS_PROJECT_ID", value: currentProject?.projectNumber || "NA"},
+                { title: "WORKS_DATE_PROPOSAL", value: Digit.Utils.pt.convertEpochToDate(currentProject?.additionalDetails?.dateOfProposal) || "NA"},
+                { title: "WORKS_PROJECT_NAME", value: currentProject?.name || "NA"},
+                { title: "PROJECT_PROJECT_DESC", value: currentProject?.description || "NA"}
+            ]
+        };
+
+        const projectDetails = {
+            title: "WORKS_PROJECT_DETAILS",
+            asSectionHeader: true,
+            values: [
+                { title: "PROJECT_LOR", value: currentProject?.referenceID || "NA" },
+                { title: "WORKS_PROJECT_TYPE", value: currentProject?.projectType ? t(`COMMON_MASTERS_${Digit.Utils.locale.getTransformedLocale(currentProject?.projectType)}`) : "NA" },
+                { title: "PROJECT_TARGET_DEMOGRAPHY", value: currentProject?.targets?.beneficiaryType?.code ? t(`COMMON_MASTERS_${currentProject?.targets?.beneficiaryType?.code}`) : "NA" }, //will update once backend makes changes for MDMS
+                { title: "PROJECT_ESTIMATED_COST", value: `₹ ${currentProject?.additionalDetails?.estimatedCostInRs}` || "NA" },
+            ]
+        };
+
+        const locationDetails = {
+            title: "WORKS_LOCATION_DETAILS",
+            asSectionHeader: true,
+            values: [
+                { title: "PROJECT_GEO_LOCATION",value: currentProject?.address?.addressLine1 || "NA" },
+                { title: "WORKS_CITY",value: currentProject?.address?.city ? t(currentProject?.address?.city) : "NA" }, //will check with Backend
```

Comment:
dont directly map city , do translate of TENANT_TENANTS_PG_CITYA logic

Reply:
done